__tests__/api.test.js

```diff
@@ -229,3 +228,116 @@ describe("POST to /api/articles/:article_id/comments", () => {
     expect(body.message).toBe("Bad Request");
   });
 });
+
+describe("PATCH to /api/articles/:article_id", () => {
+  it("should respond with status code 200 for a successful request and respond with the updated article", async () => {
+    const input = { inc_votes: 1 };
+    const articleBefore = {
```

Comment:
We probably don't need to worry about checking the article for what it looked like before and asserting what it doesn't look like, as this is inferred when we're asserting what the response does look like.